examples/backbone_marionette/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "backbone.marionette.todo.mvc",
+  "dependencies": {
+    "todomvc-app-css": "^1.0.1"
```

Comment:
You need to actually include the CSS file from this package.

Use .gitignore to make sure only the required files are included. See the backbone app UI upgrade for how to do that.